### PR TITLE
Add prune jobs command

### PR DESCRIPTION
**Pull request by soltysh**

### CODE REVIEW

File: pkg/cmd/admin/prune/jobs.go [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+	cmd := &cobra.Command{
+		Use:        name,
+		Short:      "Remove old completed and failed jobs",
+		Long:       jobsLongDesc,
+		Example:    fmt.Sprintf(jobsExample, parentName, name),
+		SuggestFor: []string{"job", "jobs"},
+		Run: func(cmd *cobra.Command, args []string) {
+			kcmdutil.CheckErr(opts.Complete(f, cmd, args, out))
+			kcmdutil.CheckErr(opts.Validate())
```
Comment (kargakis) [inline pkg/cmd/admin/prune/jobs.go]: errors returned from Validate should be wrapped as UsageError errors

Comment (soltysh) [inline pkg/cmd/admin/prune/jobs.go]: Good catch, we don't do that in other prune commands either, will fix there as well.

**Comment by mfojtik** [comment]
@soltysh I still think that this should be configurable on the controller level where I can specify a field on ScheduledJob saying how much "old" jobs I want to keep. The controller should then prune the "old" jobs automatically when a new job is scheduled.

**Comment by soltysh** [comment]
@mfojtik let's wait for what upstream will say about it, in the mean time we can introduce this command as an experimental, since SJ is still an alpha feature, wdyt?

@mfojtik for the logic itself
@openshift/api-review for api review

**Comment by soltysh** [comment]
@mfojtik apparently multi-version support bites me/us again :sob: I can't do regular `KClient.Batch().ScheduledJobs(o.Namespace).List(kapi.ListOptions{})` because it's hitting `batch/v1` instead of `batch/v2alpha1`. Not sure if that's a quick fix or something bigger...

**Comment by soltysh** [comment]
k8s 1.6 has a limit of jobs a CronJobs triggers and that will be picked with the next rebase. Closing.

**Pull request closed without merging**